### PR TITLE
chore: Move share button

### PR DESCRIPTION
**Pull request by timarney**
Fixes: https://github.com/cds-snc/platform-forms-client/issues/3059

- Moves save button out of left navigation
- Adds save button to translate page
- Adds save error toast message (also works for auto-save)

Screenshot 2024-01-10 at 2 58 41 PM

Screenshot 2024-01-10 at 3 09 37 PM

Screenshot 2024-01-10 at 3 14 04 PM

Screenshot 2024-01-10 at 3 14 52 PM

### CODE REVIEW

File: public/static/locales/fr/form-builder.json [hunk 1]
```diff
@@ -360,7 +360,7 @@
   "incomplete": "Incomplet",
   "inputDescription": "Description",
   "insertLink": "Insérer un lien  ",
-  "lastSaved": "Dernier enregistrement :",
+  "lastSaved": "updated at [fr]",
```
Comment (timarney) [inline public/static/locales/fr/form-builder.json:363]: @anikbrazeau couple of strings changed

File: public/static/locales/fr/form-builder.json [hunk 2]
```diff
@@ -713,5 +714,6 @@
     "addOther": "Ajouter un champ de saisi pour  « Autre »",
     "other": "Autre ",
     "show": "Afficher :"
-  }
-}
\ No newline at end of file
+  },
+  "saved": "Saved"
```
Comment (timarney) [inline public/static/locales/fr/form-builder.json:718]: @anikbrazeau new string here

**Comment by thiessenp-cds** [comment]
One minor change. It would be helpful to have focus placed on the "Failed - Contact support" after the error is shown. Currently looking to see how difficult that would be

**Comment by thiessenp-cds** [comment]
That could. be difficult. I think we need to refactor StyleLink to receive a ref for the error link. Then in a useEffect, if there is an error, focus that ref. There may be other ways. If we were to go this route, the code change may impact other parts of the app, so probably best to do outside this PR

**Review by thiessenp-cds — DISMISSED** [review]
We may want to revisit the Toast for this and similar use cases in a future task. Also the the comment about focussing the error link but as mentioned, that's probably best to do in a separate PR.

Nice work, LGTM :)

**Comment by timarney** [comment]
@thiessenp-cds I attempted to use a ref + use effect on the title inside the toast notification but it didn't seem to be working. Agreed we could do some further work under a future PR.

File: public/static/locales/fr/form-builder.json [hunk 3]
```diff
@@ -359,7 +359,7 @@
   "incomplete": "Incomplet",
   "inputDescription": "Description",
   "insertLink": "Insérer un lien  ",
-  "lastSaved": "Dernier enregistrement :",
+  "lastSaved": "updated at [fr]",
```
Comment (anikbrazeau) [inline public/static/locales/fr/form-builder.json:362]:
```suggestion
  "lastSaved": "mise à jour à",
```

File: public/static/locales/fr/form-builder.json [hunk 4]
```diff
@@ -712,5 +713,6 @@
     "addOther": "Ajouter un champ de saisi pour  « Autre »",
     "other": "Autre ",
     "show": "Afficher :"
-  }
-}
\ No newline at end of file
+  },
+  "saved": "Saved"
```
Comment (anikbrazeau) [inline public/static/locales/fr/form-builder.json:717]:
```suggestion
  "saved": "Enregistré"
```

File: public/static/locales/en/form-builder.json [hunk 5]
```diff
@@ -697,7 +697,8 @@
     "title": "There was an error saving the form",
     "description": "Download a copy of the form and try again or",
     "supportLink": "contact Support",
-    "errorCode": "Error code: {{code}}"
+    "errorCode": "Error code: {{code}}",
+    "failedLink": "Failed - Contact support"
```
Comment (anikbrazeau) [inline public/static/locales/en/form-builder.json:701]:
```suggestion
    "failedLink": "Saving failed - Contact support"
```

File: public/static/locales/fr/form-builder.json [hunk 6]
```diff
@@ -697,7 +697,8 @@
     "title": "Une erreur s'est produite lors de l'enregistrement du formulaire",
     "description": "Veuillez télécharger une copie du formulaire et réessayer ou",
     "supportLink": "contacter l'équipe de soutien",
-    "errorCode": "Code d'erreur : {{code}}"
+    "errorCode": "Code d'erreur : {{code}}",
+    "failedLink": "Failed - Contact support [fr]"
```
Comment (anikbrazeau) [inline public/static/locales/fr/form-builder.json:701]:
```suggestion
    "failedLink": "L'enregistrement a échoué - Contacter l'équipe de soutien"
```

**Review by thiessenp-cds — APPROVED** [review]
Nice work, LGTM :)

**Pull request merged**